contracts/rbn-staking/Redeemer.sol

```diff
@@ -0,0 +1,95 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import {ISeizer} from "../interfaces/ISeizer.sol";
+import {IVotingEscrow} from "../interfaces/IVotingEscrow.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {
+  SafeERC20
+} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
+
+contract Redeemer {
+  using SafeERC20 for IERC20;
+  using SafeMath for uint256;
+
+  // Multisig
+  address public admin;
```

Comment:
Needs the ability to transfer admin-ship, so we might as well use Ownable?

Reply:
👍